lib/views/staging-view.js

```diff
@@ -143,8 +143,10 @@ export default class StagingView {
       await this.resolutionProgressObserver.setActiveModel(this.props.resolutionProgress);
     }
 
-    const {added, removed} = compareSets(previouslySelectedItems, this.selection.getSelectedItems());
-    if (added.size > 0 || removed.size > 0) {
+    const previousSelection = Array.from(previouslySelectedItems).sort();
+    const newSelection = Array.from(this.selection.getSelectedItems()).sort();
+    const selectionChanged = !isEqual(previousSelection, newSelection);
```

Comment:
Do we need to convert these to arrays?

```javascript
a = new Set([1, 2, 3])
//=> Set(3) {1, 2, 3}
b = new Set([2, 1, 3])
//=> Set(3) {2, 1, 3}
_.isEqual(a, b)
//=> true
```

Reply:
nooope, totally unnecessary. Nice catch :)